packages/zeromq.rb

```diff
@@ -1,37 +1,28 @@
-require 'package'
+require 'buildsystems/autotools'
 
-class Zeromq < Package
+class Zeromq < Autotools
   description 'Fast messaging system built on sockets. C and C++ bindings. aka 0MQ, ZMQ.'
   homepage 'https://zeromq.org/'
-  version '4.3.2'
+  version '4.3.5'
   license 'LGPL-3'
   compatibility 'all'
-  source_url 'https://github.com/zeromq/libzmq/releases/download/v4.3.2/zeromq-4.3.2.tar.gz'
-  source_sha256 'ebd7b5c830d6428956b67a0454a7f8cbed1de74b3b01e5c33c5378e22740f763'
-  binary_compression 'tar.xz'
+  source_url 'https://github.com/zeromq/libzmq.git'
+  git_hashtag "v#{version}"
+  binary_compression 'tar.zst'
 
   binary_sha256({
-    aarch64: '203bef19128bd37bb9320c1413693edaad122804276cb3114e540ab0a77dae3f',
-     armv7l: '203bef19128bd37bb9320c1413693edaad122804276cb3114e540ab0a77dae3f',
-       i686: '9695e614855a6edb5d0c5ac190ee4dbd6b49dccd6f235f9bc93ffcc70f0e4f9f',
-     x86_64: '531fbc84e6d4d4572ae1b4e5e5e84ca63303151d9674cc0e0f79edece7a27d51'
+    aarch64: '0c9dd94783114203775f59f38caf4166c00827c2b66b66b2e1812e4a595a6be3',
+     armv7l: '0c9dd94783114203775f59f38caf4166c00827c2b66b66b2e1812e4a595a6be3',
+       i686: 'c4c2847266ee5503a4ef92850d35ed3c7defa75070c7cd354ab90ca2efe60def',
+     x86_64: 'e137a13cbbe23708483daaf1800dc789b792555e89213e87247b69490228e6f5'
   })
 
+  depends_on 'libbsd'
+  depends_on 'libmd' # R
   depends_on 'libunwind'
 
-  def self.build
-    system './configure',
-           "--prefix=#{CREW_PREFIX}",
-           "--libdir=#{CREW_LIB_PREFIX}",
-           '--disable-dependency-tracking'
-    system 'make'
-  end
-
-  def self.check
-    system 'make', 'check'
-  end
-
-  def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-  end
+  # warning: failed to load external entity "http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl"
+  # compilation error: file /tmp/xmlto-xsl.bNfymk line 4 element import
+  configure_options '--without-docs'
```

Comment:
We should try to restore the man pages, if possible.  Rework docbook packages?

Reply:
This has been an issue for as long as I can remember-- I'm leaving this comment here so that I can find it when we fix it, but that's a bigger issue for later.

Reply:
See #10376.

Reply:
I don't have time to rework the doc book packages. Consider opening an issue for that and asking for help for it...